### PR TITLE
Rewire local

### PR DESCRIPTION
**Pull request by converge**
This PR:

- keeps old local.go command
- adds a new local.go command
- structure the three different installation moment into functions (pre install, install, post install)
- use single source of truth for flags
- organize local into local package
- remove the conditionals to check if its local install or not

### CODE REVIEW

File: cmd/local/local.go [hunk 1]
```diff
@@ -0,0 +1,483 @@
...
+	localCmd.Flags().BoolVar(&dryRun, "dry-run", false, "set to dry-run mode, no changes done on cloud provider selected")
+	localCmd.Flags().BoolVar(&silentMode, "silent", false, "enable silentMode mode will make the UI return less content to the screen")
+	// todo: get it from GH token , use it for console
+	localCmd.Flags().StringVar(&adminEmail, "admin-email", "", "the email address for the administrator as well as for lets-encrypt certificate emails")
+
+	localCmd.Flags().StringVar(&metaphorBranch, "metaphor-branch", "main", "metaphro application branch")
+	localCmd.Flags().StringVar(&gitOpsBranch, "gitops-branch", "main", "version/branch used on git clone - former: version-gitops flag")
+	localCmd.Flags().StringVar(&gitOpsRepo, "gitops-repo", "gitops", "")
+
+	localCmd.Flags().BoolVar(&enableConsole, "enable-console", true, "If hand-off screen will be presented on a browser UI")
```
Comment (6za) [inline cmd/local/local.go:65]: this will disable the support of the config file, right?

Comment (6za) [reply cmd/local/local.go:65]: Or do we plan to mix global vars with the current read config logic?

**Review by 6za — APPROVED** [review]

**Comment by 6za** [comment]
let merge and see.

File: cmd/local/local.go (hunk 1 above)
Comment (converge) [reply cmd/local/local.go:65]: I'll tackle that after merging

**Pull request merged**